src/fragments/lib/logging/common/sending_logs/sending_logs.mdx

```diff
@@ -0,0 +1,20 @@
+The Amplify library already uses the Amplify Logger internally.  Each Amplify feature logs messages at various log levels.  You can log your own custom messages and various log levels depending on your use cases.  Only log levels that are the same or below the log level defined in the logging constraints are sent to Amazon CloudWatch.
+
+## Log an message
+Get an instance of the `Logger` and specifying a `category name` and/or `namespace`.  Use the `Logger` instance to log messages at the desired log level.  The `category name` and `namespace` values are used to tag your log messages that will appear in CloudWatch.  It is also use to identify and fetch the logger again without creating a new `Logger` instance.
+
+import iosSendLog from '/src/fragments/lib/logging/ios/sending_logs/10_sending_logs.mdx';
+import androidSendLog from '/src/fragments/lib/logging/android/sending_logs/10_sending_logs.mdx';
+
+<Fragments
```

Comment:
can we have the code snippet actually use a JSON object and pass it to the logger?

Reply:
Updated sample text to use a json string message.

Reply:
so I would have to pass a string at the end of the day? Can we maybe show the example be an actual types json that we then change toString() - since that is how I would imagine a developer would interact with it in code.

Reply:
I've updated the example code.